articles/flutter-vs-reactnative/index.md

```diff
@@ -0,0 +1,113 @@
+
+#### Take 2.5 - Enter React Native
+React is a front end framework for the web. What React Native did was extend the functionality of React to native Android and iOS platforms. The developer got to code the UI as React components.
+
+The developer could use the same code used in their React web app for some UI elements. Later React Native would replace those web elements with the corresponding native elements. For example, `<span>` primitive is replaced with TextView in the native build and UI View in iOS build.
+
+This was better than the earlier approach because:
+1. Web developers could dive into mobile app development without a steep learning curve.
+2. Code can be reused to a large extent.
+3. It offered better performance than a web app wrapper as it ran closer to metal.
+
+#### Take 3: A Blank canvas
+Let's take a look at mobile game development frameworks like [Unity](https://unity.com/). They don't have anything like the Native UI component. All they provide is a blank 2D canvas or an empty 3D space. You are free to show anything you want in that space, irrespective of the mobile platform you are on.
+
+Flutter works the same way, unlike other app development frameworks. The platform is just concerned with rendering a canvas (Skia canvas). Once the canvas is created, Flutter paints on that canvas 60 times a second.
+
+That way it does not have to deal with platform-specific components. What prevents Flutter from supporting another platforms is its inability to run the [Skia canvas](https://skia.org/user/api/skcanvas_overview) on that platform.
+
+#### What Flutter gets right
+##### Hot Reload - The MVP of Flutter
+Flutter hot reloads are super fast. Developers can make changes and view them on the fly within milliseconds. Want to try out which color suits the app? You can keep changing the color and it’ll show up in the app near instantly! This is great for rapid prototyping and creating an MVP.
+
+##### 1 codebase - multiple platforms
+Painting on canvas, makes the user feel like the Bob Ross of while working with frameworks! Due to the all the platforms that exist for users.
+
+A user can work with:
+- Android
+- iOS,
+- Web
+- Windows
+- MacOS
+- Linux
+- Smartwatches
+- IoT.
+
+##### Smooth UI, rendered at 60fps
+Flutter works more like a gaming engine, than an app framework you may be used to. As already explained, the UI is built and rendered on a Skia Canvas as it changes. Flutter updates the UI at 60fps, and uses the GPU for most of the work.
+
+While this doesn’t affect how we build an app, it is the reason the Flutter UI is buttery smooth. Dart based application code will continue to run via the CPU, and in the specialized UI Thread, when touching UI related components.
+
+##### A UI first framework
```

Comment:
this paragraph could be reworded - 
I believe this may be important to the article: "A UI first framework"